src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/entity/Referral.kt

```diff
@@ -53,9 +53,9 @@ data class Referral(
 
   @Column val releaseDate: LocalDate? = null,
 
-  @Column(nullable = false) val descriptionOfConcern: String,
+  @Column(nullable = false) val descriptionOfConcern: String?,
 
-  @Column(nullable = false) val knownReasons: String,
+  @Column(nullable = false) val knownReasons: String?,
```

Comment:
are we okay to keep nullable=false annotation for these now-nullable columns?

Reply:
Good spot, thanks.